package.json

```diff
@@ -47,6 +47,7 @@
   },
   "peerDependencies": {
     "immutable": "^3.7.6",
+    "postinstall-build": "^5.0.0",
```

Comment:
I think I finally got that part working. The namespace changes of the package caused issues among other things.

This was meant to go into the dependencies as a way to directly include the github repo and have it temporarily do an install and a compile before cleaning up. 

I had to fork that repo because of the namespace for this package. I also initiated a pull request from them to include my bugfix for namespaced packages https://github.com/exogen/postinstall-build/pull/26 .